server/src/main/kotlin/delta/codecharacter/server/auth/oauth2/CustomOAuth2SuccessHandler.kt

```diff
@@ -37,14 +39,17 @@ class CustomOAuth2SuccessHandler(@Lazy @Autowired private val authService: AuthS
             val loginType = LoginType.valueOf(provider ?: "GOOGLE")
             try {
                 val token = authService.oAuth2Login(email, loginType)
+                TimeZone.setDefault(TimeZone.getTimeZone("GMT"))
```

Comment:
Don't set JVM default timezone, use some instance method to set timezone instead. See [this](https://stackoverflow.com/a/9891971/9224581).